### PR TITLE
[Discover] Fixes safari overflow bug

### LINKED ISSUE #5941: [BUG] Rendering error of Discover in Safari for OpenSearch v 2.12
In Safari on a Macbook (Safari: Version 17.3.1 (19617.2.4.11.12)), when I go to the OpenSearch playground Discover and change the time range to 60 minutes, it navigates to a URL like this:
https://playground.opensearch.org/app/data-explorer/discover#?_a=(discover:(columns:!(_source),isDirty:!f,sort:!()),metadata:(indexPattern:ff959d40-b880-11e8-a6d9-e546fe2bba5f,view:discover))&_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-60m,to:now))&_q=(filters:!(),query:(language:kuery,query:''))

The result is a rendering error that looks like this:
image

### PR DESCRIPTION
**Pull request by ashwin-pc**
### Description

The translate prop on the source cell causes a glitch in Safari that causes the overflow property to not work as expected in some scenarios. Removing it fixes the issue.

### Issues Resolved

fixes #5941

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by seraphjiang** [comment]
> The translate prop on the source cell causes a glitch in Safari that causes the overflow property to not work as expected in some scenarios. Removing it fixes the issue.

could we share the screenshoot before and after to help review the PR

**Review by ananzh — DISMISSED** [review]

**Comment by ashwin-pc** [comment]
@seraphjiang I did not add a screenshot here since the issue makes it obvious what the bug is and the fix basically get rid of the bug. There is no visual change to the UI because of this PR. I can add it if it helps though

File: CHANGELOG.md [hunk 1]
```diff
@@ -71,6 +71,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [BUG][Multiple Datasource] Fix datasource testing connection unexpectedly passed with wrong endpoint [#5663](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5663)
 - [Table Visualization] Fix filter action buttons for split table aggregations ([#5619](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5619))
 - [BUG][Discover] Allow saved sort from search embeddable to load in Dashboard ([#5934](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5934))
+- [BUG][Discover] Fix safari overflow bug ([#5948](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5948))
```
Comment (AMoo-Miki) [inline CHANGELOG.md:74]:
```suggestion
- [Discover] Fix safari overflow bug ([#5948](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5948))
```

**Review by ananzh — APPROVED** [review]

**Review by AMoo-Miki — DISMISSED** [review]

**Review by ananzh — APPROVED** [review]

**Review by AMoo-Miki — APPROVED** [review]

**Comment by ananzh** [comment]
@seraphjiang before the fix, in Safari 17 it looks like this (2.11 test domain)
Screenshot 2024-02-26 at 4 00 40 PM

After this, just normal display.

**Pull request merged**